moodlecli/main.py

```diff
@@ -250,3 +250,33 @@ def export_users_by_course(source_course_id, bucket_name, key):
     )
 
     aws.put_json_data(users, bucket_name, key)
+
+
+@cli.command()
+@click.argument('bucket_name')
+@click.argument('directory')
+def export_users_bulk(bucket_name, directory):
+    "Output the users for a series of classes to an s3 bucket"
+    moodle = get_moodle_client()
+
+    course_data = moodle.get_courses()
+    course_ids = utils.get_course_ids(course_data)
+    for id in course_ids:
+        user_data = moodle.get_users_by_course(id)
+        key = f'{directory}/users_course-{id}.json'
```

Comment:
I believe [here](https://github.com/openstax/k12/issues/41#issuecomment-1034273738) we talked about using a slightly different convention where I think `directory` would be something like `algebra1/spring2022/moodle/users` and the key would be `algebra1/spring2022/moodle/users/{id}.json` (similarly for the other command in this PR it would be  `algebra1/spring2022/moodle/grades` and `algebra1/spring2022/moodle/grades/{id}.json`). It's okay if we want to revisit that, I just want to check if this shift is intentional.

Reply:
Right, right, right. This had slipped my mind yesterday - This way we can mandate this convention and remove the directory command.

Reply:
Oh, I think we'll still want the directory parameter so the user can specify the `{content}/{semester}/moodle/users` piece, no? Or are you thinking we just pass the course and semester as arguments and then the CLI codifies the convention using those values?

Reply:
Ah, I was under the impression that the content and the semester could be extracted from the API and each could be saved - but i realize content is different than course. No, I agree, I think that the user defining the directory will provide the admin the most flexibility. However I don't think they'll need to specify the key/file_name. Do you agree that we could just generate those based on ID?